dbms/src/Storages/DeltaMerge/WriteBatches.h

```diff
@@ -42,12 +42,13 @@ struct WriteBatches : private boost::noncopyable
     WriteLimiterPtr write_limiter;
 
     WriteBatches(StoragePool & storage_pool_, const WriteLimiterPtr & write_limiter_ = nullptr)
-        : log(storage_pool_.getNamespaceId())
-        , data(storage_pool_.getNamespaceId())
-        , meta(storage_pool_.getNamespaceId())
-        , removed_log(storage_pool_.getNamespaceId())
-        , removed_data(storage_pool_.getNamespaceId())
-        , removed_meta(storage_pool_.getNamespaceId())
+        : ns_id(storage_pool_.getNamespaceId())
```

Comment:
So it is because we forget to init the `ns_id` var?😂

Reply:
Verified. it's the problem.